homeassistant/components/device_tracker/phonetrack_oc.py

```diff
@@ -0,0 +1,61 @@
+"""
+Support for PhoneTrackOC device tracking.
+"""
```

Comment:
Please follow the style of other files for the file header aka add a link to the documentation.